model/uzivatel.php

```diff
@@ -942,6 +942,22 @@ protected static function nactiUzivatele($where)
     return $uzivatele;
   }
 
+  /**
+   * Změní údaj aktuálního uživatele.
+   * @param array $udaj
+   */
+  function zmenitUdaj(array $udaj) {
```

Comment:
Tuto metodu pls nepřidávej. Ať to zůstane u řešení, že v adminu se zavolá Uzivatel::cisloOp, pokud se mění občanka, a jinak přímo ten dbUpdate, který tam byl.

Že se z adminu přímo šahá do DB není dobře, ale přidáním metody "změň libovolný sloupec" do modelu se to nevyřeší. Zapouzdření se tím tak či tak ztrácí a u přidání do modelu vidím riziko, že to bude svádět lidi používat takovou metodu na víc místech.

Finální "správné" řešení zatím nevím (bude to nějak souviset s návrhem formulářů), nicméně pro teď to asi musíme řešit plánem výš.